docs/FTP-batchsource.md

```diff
@@ -17,28 +17,41 @@ Properties
 **Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
 
 **Path:** Path to file(s) to be read. The path uses filename expansion (globbing) to read files.
-Path is expected to be of the form prefix://username:password@hostname:port/path (Macro-enabled)
+Path is expected to be of the form prefix://username:password@hostname:port/path
 
-**Regex Path Filter:** Regex to filter out files in the path. It accepts regular expression which is applied to the complete
-path and returns the list of files that match the specified pattern. (Macro-enabled)
+**Format:** Format of the data to read.
+The format must be one of 'blob', 'csv', 'delimited', 'json', 'text', 'tsv', or the
+name of any format plugin that you have deployed to your environment. Note that FTP does
+not support seeking in a file, so formats like avro and parquet cannot be used.
+If the format is a macro, only the formats listed above can be used.
+If the format is 'blob', every input file will be read into a separate record.
+The 'blob' format also requires a schema that contains a field named 'body' of type 'bytes'.
+If the format is 'text', the schema must contain a field named 'body' of type 'string'.
 
-**File System Properties:** A JSON string representing a map of properties
-needed for the distributed file system. (Macro-enabled)
+**Get Schema:** Auto-detects schema from file. Supported formats are: csv, delimited, tsv, blob and text.
 
-**Allow Empty Input:** Identify if path needs to be ignored or not, for case when directory or file does not
-exists. If set to true it will treat the not present folder as 0 input and log a warning. Default is false.
+Blob - is set by default as field named 'body' of type bytes.
+
+Text - is set by default as two fields: 'body' of type bytes and 'offset' of type 'long'.
+
+JSON - is not supported, user has to manually provide the output schema.
```

Comment:
Can this be two sentences?  JSON - is not supported. You must manually provide the output schema.